query-engine/query-engine-c-abi/src/migrations.rs

```diff
@@ -142,9 +142,9 @@ pub fn list_migrations(database_filename: &Path) -> Result<Vec<MigrationRecord>>
         let failed_at: Option<Timestamp> = row.get(4).unwrap();
 
         entries.push(MigrationRecord {
-            id,
+            _id: id,
```

Comment:
Left these two for now to be safe in case they were intentionally left for future but AFAICT they are currently useless and can be removed from both the struct and the SQL query.